antiplag/serializers.py

```diff
@@ -45,24 +45,28 @@ class Meta:
 
 
 class ResultSerializer(serializers.ModelSerializer):
+    id = serializers.CharField(source="match_id")
+    name = serializers.CharField(source="match_name")
+
     class Meta:
         model = Result
-        fields = ("percentage", "matched_docs")
+        fields = ("id", "name", "percentage")
```

Comment:
Don't we need `ranges` from `matched_docs` to be serialized as well?

Reply:
AFAIK, not at the moment. Actually, we use it - those ranges are are used in document property - those flattened and sorted, but we don't provide it to client.